### PR TITLE
Add Efinity and Westfinity SS58 address prefix

### PR DESCRIPTION
**Pull request by coinfork**
Hello everyone,

The Enjin team would like to add our SS58 prefixes for Efinity (Polkadot) and Westfinity (Westend).
Project website: efinity.io

Thank you!

### CODE REVIEW

File: ss58-registry.json [hunk 1]
```diff
@@ -729,6 +729,24 @@
       "decimals": [18],
       "standardAccount": "*25519",
       "website": "https://ata.network"
+    },
+    {
+      "prefix": 1110,
```
Comment (joepetrowski) [inline ss58-registry.json:734]: Not in order

File: ss58-registry.json [hunk 2]
```diff
@@ -729,6 +729,24 @@
...
+      "website": "https://efinity.io/"
+    },
+    {
+      "prefix": 8688,
+      "network": "westfinity",
+      "displayName": "Westfinity: Efinity Testnet",
+      "symbols": ["WFI"],
+      "decimals": [18],
+      "standardAccount": "Sr25519",
+      "website": "https://efinity.io/"
```
Comment (joepetrowski) [inline ss58-registry.json:749]: I don't think testnets should have a prefix reserved

**Review by joepetrowski — CHANGES_REQUESTED** [review]

**Comment by BenWhiteJam** [comment]
@coinfork any update on the comments?

**Comment by coinfork** [comment]
> @coinfork any update on the comments?

Fixed!

**Review by joepetrowski — APPROVED** [review]

**Review by BenWhiteJam — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Pull request merged**